src/test/java/com/es/phoneshop/model/entity/ArrayListProductDaoTest.java

```diff
@@ -1,5 +1,9 @@
-package com.es.phoneshop.model.product;
+package com.es.phoneshop.model.entity;
```

Comment:
Your test should have the same package path as a tested class.
**model/dao/impl**